### PR TITLE
regard `SHOW` as sql command for mysql, postgres protocol

### PR DESCRIPTION
**Pull request by chengjoey**
### What does this PR do?
network ebpf mysql, postgres protocol regard `show` as sql command

### CODE REVIEW

File: releasenotes/notes/regrad-show-as-sql-command-e08c0a29ee46d396.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Ebpf network regard show as sql command for mysql, postgres protocol
```
Comment (rtrieu) [inline releasenotes/notes/regrad-show-as-sql-command-e08c0a29ee46d396.yaml:4]:
```suggestion
    eBPF networks regard show as a SQL command for MySQL and Postgres protocols
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some formatting feedback.

File: releasenotes/notes/regrad-show-as-sql-command-e08c0a29ee46d396.yaml (hunk 1 above)
Comment (chengjoey) [reply releasenotes/notes/regrad-show-as-sql-command-e08c0a29ee46d396.yaml]: done~

**Comment by guyarb** [comment]
Hey @chengjoey 👋🏻 

First of all, thank you for submitting this change, we appreciate the effort!
However, we intentionally decided to keep the list of commands limited as our overall code is pretty complex, which gives us very small room for modifications, as a more complicated code can lead us to reach to the eBPF verifier limitations (on number of branches in the code or number of instructions). We chose the most common commands to be added to the list, and unfortunately, `show` wasn't one of them. Furthermore, the code lives in our classification code, and for that, it is suffice for us to find a single matched command in the connection to classify it as postgres or as mysql, so the impact of unsupported commands (like show, begin, commit, etc) is narrowed down.

**Pull request closed without merging**